perun-web-gui/src/main/java/cz/metacentrum/perun/webgui/json/tasksManager/GetFacilityServicesState.java

```diff
@@ -246,6 +246,9 @@ else if (Arrays.asList("GENERATING","GENERATED","SENDING").contains(row.getStatu
 				else if (row.getStatus().equalsIgnoreCase("DONE")){
 					return "rowgreen";
 				}
+				else if (row.getStatus().equalsIgnoreCase("WARNING")){
+					return "rowgreenyellow";
```

Comment:
We discussed color implications and we want to make it more like a semaphore, so that warning would be visible, but not so much outstanding like now. Basically styles should be as follows:

**Task**
```
WAITING = empty (white - table default)
PLANNED = rowlightyellow
GENERATING, GENERATED, SENDING = rowyellow
DONE = rowgreen
WARNING = roworange (this is added)
GENERROR, SENDERROR, ERROR = rowred
```
On some places in GUI, where "rowred" style is used for the Task, it is compared only to the ERROR, but should support other states (GENERROR/SENDERROR). Please find them and update them. Thank you.

**TaskResult**
```
DONE = rowgreen
DENIED = rowyellow (while I don't recall ever seeing TaskResult in a denied state).
WARNING = roworange (this is changed)
ERROR = rowred (this is changed from orange to red)
```
As we never ever used ``FATAL_ERROR`` from ``TaskResultStatus`` we can remove it from the object in core and here in GUI and use its color for the error state as mentioned above.

You can then remove newly added ``rowgreenyellow`` styles from CSS.